control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -903,7 +903,6 @@ func (r *reconciler) reconcileCloudCredentialSecrets(ctx context.Context, hcp *h
 		}
 		for arn, secret := range map[string]*corev1.Secret{
 			hcp.Spec.Platform.AWS.RolesRef.IngressARN:       manifests.AWSIngressCloudCredsSecret(),
-			hcp.Spec.Platform.AWS.RolesRef.StorageARN:       manifests.AWSStorageCloudCredsSecret(),
```

Comment:
Please advise if removing this is correct.

Reply:
@bertinatto I think this is actually the line that causes there is no `ebs-cloud-credentials` Secrets for AWS EBS CSI driver in the guest cluster.

Reply:
This PR basically stops creating the secret in the guest (the line above) and starts creating it in the mgmt cluster.
Until CSO is moved to the mgmt cluster we need to render the secret in **both** clusters. And the clean the guest as a subsequent PR.

Reply:
Good catch. Commented this out.